packages/core/src/validator/offchain/offchain-tx-validator.ts

```diff
@@ -171,6 +171,13 @@ export class OffchainTxValidator extends OffchainValidatorContext
         utxoRoot: inclusionRef.toString(),
       },
     })
+    // If any of the found header is slashed, it returns false
+    const slashed = await this.layer2.db.findMany('Slash', {
+      where: {
+        hash: headers.map(h => h.hash),
+      },
+    })
+    if (slashed.find(p => p.finalized === true)) return false
```

Comment:
I think this line is already processing the slashed block well.
https://github.com/zkopru-network/zkopru/blob/9f0719b97e2973620414e0e018903f6751571249/packages/core/src/validator/offchain/offchain-tx-validator.ts#L197-L199

Querying slash data seems a duplicated job. Let me close this PR